ansible/roles/stack-sunbird/templates/sunbird_player.env

```diff
@@ -229,7 +229,9 @@ cloud_service_provider={{cloud_service_provider}}
 cloud_private_storage_accountname={{cloud_private_storage_accountname | default("")}}
 cloud_private_storage_secret={{cloud_private_storage_secret | default("")}}
 cloud_private_storage_region={{cloud_private_storage_region | default("ap-south-1")}}
+cloud_private_storage_region={{cloud_private_storage_region | default("ap-south-1")}}
 cloud_private_storage_project={{cloud_private_storage_project | default("")}}
+cloud_private_storage_endpoint={{cloud_private_storage_endpoint | default("")}}
```

Comment:
@ddevadat looks like cloud_private_storage_region variable is repeated here. We can remove it

Reply:
removed duplicate entry